tests/istio.io/README.md

```diff
@@ -6,12 +6,66 @@ This folder contains tests for the different examples on Istio.io.
 
 ## Writing Tests
 
+Tests in the Istio.io framework support the following test types:
+
+* Executing scripts using the `RunScript(script string, output outputType, verifier VerificationFunction)` method
+This method accepts an argument which is the name of a script to run, a desired outputType (`TextOutput`,`YamlOutput`, and `JsonOutput`), and an optional verification function. If verifier is set to nil, then any errors from the script will be interpreted as a failure of that step.
+* Applying/Deleting files using `Apply(namespace string, path string)`/`Delete(namespace string, path string)`
```

Comment:
What does "apply" mean in this context?

Reply:
Thanks. Cleaned up the description to say: 

Use Apply(namespace string, path string) to execute a kubectl apply -f with the specifed file on the namespace specified. Use Delete(namespace string, path string) to do a kubectl delete -f. In both cases, the path is relative to the root of the Istio source tree.